helm/charts/kratos/templates/deployment.yaml

```diff
@@ -43,9 +43,7 @@ spec:
     {{- end }}
     {{- if .Values.deployment.extraInitContainers}}
       initContainers:
-        {{- if .Values.deployment.extraInitContainers }}
```

Comment:
I know that this if is redundant (k8s will remove the empty field), but for consistency please either remove it in other places too, or keep it :)

Reply:
Wasn't sure what other places you were referring to (I couldn't find any) so I added it back!